code/modules/events/spider_terror.dm

```diff
@@ -0,0 +1,28 @@
+
+/datum/event/spider_terror
+	announceWhen	= 400
```

Comment:
don't do this, tabbing properties to match assignment operators
just do a space before and after the assignment operator
`announceWhen = 400`